src/store/mutations.js

```diff
@@ -39,14 +37,23 @@ const addFolderToState = (state, account) => folder => {
 	return id
 }
 
+const sortAccounts = accounts => {
+	accounts.sort((a1, a2) => a1.order - a2.order)
+	return accounts
+}
+
 export default {
 	savePreference(state, {key, value}) {
 		Vue.set(state.preferences, key, value)
 	},
 	addAccount(state, account) {
 		account.collapsed = true
 		Vue.set(state.accounts, account.id, account)
-		Vue.set(state, 'accountList', sortBy(identity)(state.accountList.concat([account.id])))
+		Vue.set(
+			state,
+			'accountList',
+			sortAccounts(state.accountList.concat([account.id]).map(id => state.accounts[id])).map(a => a.id)
```

Comment:
a bit hard to understand all those functions.
Adding a step with a const and some comments would helps a lot for this file :)